### PR TITLE
Add "Hello world kubeflow pipelines" tutorial

### PR DESCRIPTION
**Pull request by ptitzler**
This PR adds a new tutorial that outlines how to create a notebook pipeline and run it on Kubeflow Pipelines. This tutorial complements the "Hello World" tutorial, which described the tasks for local pipeline execution.

### CODE REVIEW

File: pipelines/hello_world_kubeflow_pipelines/README.md [hunk 1]
```diff
@@ -0,0 +1,249 @@
...
+![Run a notebook pipeline locally or on Kubeflow Pipelines](doc/images/notebook_pipeline_local_and_remote.png)
+
+In this tutorial you will learn how to create a notebook pipeline and run it on Kubeflow Pipelines. Take a look at the [Hello World tutorial](https://github.com/elyra-ai/examples/tree/master/pipelines/hello_world) to learn how to run notebook pipelines locally in JupyterLab. 
+
+### Prerequisites
+
+To complete this tutorial you need
+- [JupyterLab 2.x with the Elyra extension v1.1 (or newer) installed](https://elyra.readthedocs.io/en/latest/getting_started/installation.html)
+- Access to a [Kubeflow Pipelines](https://www.kubeflow.org/docs/pipelines/overview/pipelines-overview/) deployment
+
```
Comment (lresende) [inline pipelines/hello_world_kubeflow_pipelines/README.md:18]: Do you want to mention that, if they don't have access to KFP, they could deploy one locally if they have the resources? https://elyra.readthedocs.io/en/latest/recipes/deploying-kubeflow-locally-for-dev.html

**Comment by lresende** [comment]
I still think that, for visibility purposes, having the actual tutorial on the docs page would make it more visible

File: pipelines/hello_world_kubeflow_pipelines/README.md (hunk 1 above)
Comment (ptitzler) [reply pipelines/hello_world_kubeflow_pipelines/README.md:18]: Yep, will do.

**Comment by ptitzler** [comment]
Closes #12 

> I still think that, for visibility purposes, having the actual tutorial on the docs page would make it more visible. I mean, the main md file there, and pointing to everything else here.

I agree. Will deliver that in a different PR since it's another repo. Will add reference to PR.

File: pipelines/hello_world_kubeflow_pipelines/README.md (hunk 1 above)
Comment (ptitzler) [reply pipelines/hello_world_kubeflow_pipelines/README.md:18]: done

**Comment by ptitzler** [comment]
Ah sorry, I believe I have misinterpreted your earlier comment

> I still think that, for visibility purposes, having the actual tutorial on the docs page would make it more visible. I mean, the main md file there, and pointing to everything else here.

image

You are saying under "Tutorials" (somewhere in the doc tree) we would publish the actual markdown and images and only the tutorial artifacts (notebooks etc) would reside in the `examples` repository, right? I do agree having an anchor there is something we should do but my concern is about having the actual instructions live _only there_. 

#### Approach 1: instructions  and artifacts live in `examples` repository (original proposal in this PR)
  - Advantages: 
    - Everything the user needs is in a single repository. By cloning the repo all assets are available "locally" -  on the machine where the repo was cloned. The user doesn't need to open the documentation URL to access the instructions.
 - Disadvantages:
     - The tutorials don't get much exposure and are therefore hard to find
    - The look-and-feel ("the rendering") of the tutorials is not similar to the documentation

#### Approach 2: instructions live in the doc; artifacts live in `examples` repository
  - Advantages: 
    - The tutorials get higher visibility because they are part of the documentation.
    - The look-and-feel of the tutorials is similar to the documentation
 - Disadvantages:
     - A user would need to clone two repositories to gain local access to all assets (instructions and artifacts)

#### Approach 3: a stub (short summary) for each tutorial lives in the documentation; instructions and artifacts live in the `examples` repository
  - Advantages: 
    - The tutorials get higher visibility because they are part of the documentation.
    - Everything the user needs is in a single repository. By cloning the repo all assets are available "locally" -  on the machine where the repo was cloned. The user doesn't need to open the documentation URL to access the instructions.
 - Disadvantages:
     - The look-and-feel of the tutorials is not similar to the documentation

#### Approach 4: The instructions and doc artifacts live in the documentation and the `examples` repository.
  - Advantages: 
    - The tutorials get higher visibility because they are part of the documentation.
    - Everything the user needs is in a single repository. By cloning the repo all assets are available "locally" -  on the machine where the repo was cloned. The user doesn't need to open the documentation URL to access the instructions. 
 - Disadvantages:
     - Lots of duplication and harder to maintain

**Comment by ptitzler** [comment]
We've discussed this in today's community meeting and will  proceed with approach 3. The doc change will be delivered in a PR for https://github.com/elyra-ai/elyra/issues/905

**Pull request merged**